static/app/views/settings/projectAlerts/settings.tsx

```diff
@@ -166,18 +147,14 @@ function ProjectAlertSettings({canEditRule, params}: ProjectAlertSettingsProps)
 
           {canEditRule && (
             <PluginList
-              organization={organization}
               project={project}
               pluginList={(pluginList ?? []).filter(
                 p => p.type === 'notification' && p.hasConfiguration
               )}
-              onDisablePlugin={handleDisablePlugin}
```

Comment:
No longer passing a callback since `useTogglePluginMutation` optimistically updates & invalidates the cached values for plugins list & plugin details